pages/serverless-containers/how-to/add-trigger-to-a-container.mdx

```diff
@@ -39,6 +38,12 @@ You can create triggers on private containers, but to update the privacy of a co
 
 ## Queue triggers
 
+<Message type="note">
+You must have at least `MessagingAndQueuingReadOnly` permissions in the project containing the queue to add a queue trigger to Serverless Containers.
```

Comment:
```suggestion
You must have at least the `MessagingAndQueuingReadOnly` permission in the project containing the queue to add a queue trigger to Serverless Containers.
```

Reply:
permission or permission set? I never know

Reply:
I think permission set is more accurate
Also Project with capital P!